src/main/kotlin/org/jitsi/rtp/Packet.kt

```diff
@@ -31,4 +34,21 @@ abstract class Packet(
         otherTypeCreator(buffer, offset, length);
 
     public abstract override fun clone(): Packet
+
+    private fun getPacketType(buf: ByteArray, offset: Int): Int = buf.get(offset + 1).toPositiveInt()
```

Comment:
could we do these in an `RtpProtocol` type helper class? we've got other types we derive from packet (e.g. DtlsPacket).

Reply:
Great. We could have looksLikeDtls :)

Reply:
sure, but maybe for organization it'd be nice to define these as extension functions in another file--they feel like 'addons' to Packet, not core functionality, so i think it'd be nice to stick them elsewhere...some packet protocols extensions file or something?